omod/src/main/webapp/pages/metadata/locations/locationAttributeType.gsp

```diff
@@ -100,7 +100,7 @@
             formFieldName: "name",
             id           : "name",
             maxLength    : 101,
-            initialValue : ui.encodeHtmlContent((locationAttributeType.name ?: ''))
```

Comment:
Yeah, that was a deliberate decision. Here is my reasoning:

ui.encodeHtmlContent will ensure attackers cannot inject `<script></script>` tags into html, so it is suitable for data that appears within `<textarea>` or `<td>` tags.

On the other hand, ui.encodeHtmlAttribute will ensure attackers cannot inject quotation characters, which would allow them to add additional html attributes. E.g.:

`<input id="name-field" value="${ it.location.name }" >`
when name is `" onmouseover="alert('xss')` becomes
`<input id="name-field value="" onmouseover="alert('xss')">`